hardware/sim/README.md

```diff
@@ -1,12 +1,15 @@
-# Building a model
+# Simulation
+SNAP supports *Xilinx xsim* and *Cadence irun* tools for simulation.
+The environment variable `SIMULATOR` (see make snap_config) selects the simulator and can be set to `xsim` or to `irun`.  
+
+## Building a model
 see [../README.md](../README.md) for further instructions
 Note: The Makefile for building a model calls Vivado to export a script `top.sh` for compile & elaborate & simulate.
 By default this already calls the simulator. In SNAP the simulator execution is disabled and replaced by [./run_sim](./run_sim).  
```

Comment:
The recommendation should be to call `make sim` from the SNAP root directory instead or calling `run_sim` directly